example_test.go

```diff
@@ -237,3 +238,57 @@ func ExampleCredential() {
 		panic(err)
 	}
 }
+
+func ExampleTrace() {
```

Comment:
This example won't show up on the doc website.

Reply:
Moved this example into the trace package.